CEXT-4264: Add the possibility of disabling shipping methods (#327)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/use-cases.md

You can register multiple webhooks for different shipping methods or shipping carriers by adding them into the same batch to ensure they are executed in parallel or create multiple batches to execute them sequentially.

### Remove shipping method

> The `plugin.magento.out_of_process_shipping_methods.api.shipping_rate_repository.get_rates` webhook within adding shipping methods allows you to remove specific shipping methods from the list of available options

Reviewer: ```suggestion
The `plugin.magento.out_of_process_shipping_methods.api.shipping_rate_repository.get_rates` webhook allows you to remove specific shipping methods from the list of available options.
```

> For example, if you are using the `flatrate` shipping method but certain conditions require it to be disallowed, you need to update your webhook response to mark the shipping method as removed. You can find an example in [`actions/shipping-methods.js`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/shipping-methods/index.js).

Reviewer: ```suggestion
If you use the `flatrate` shipping method, but want to disable it, you must update your webhook response to mark the shipping method as removed. This example is demonstrated in [`actions/shipping-methods.js`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/shipping-methods/index.js).
```
